mmdet/apis/inference.py

```diff
@@ -212,6 +212,7 @@ async def async_inference_detector(model, imgs):
 def show_result_pyplot(model,
                        img,
                        result,
+                       palette=None,
                        score_thr=0.3,
                        title='result',
                        wait_time=0):
```

Comment:
Avoid break using positional arguments.
```suggestion
def show_result_pyplot(model,
                       img,
                       result,
                       score_thr=0.3,
                       title='result',
                       wait_time=0,
                       palette=None,):
```

Reply:
Thank you for your suggestion.